### PR TITLE
General - Move to HEMTT v1.0

### PR DESCRIPTION
**Pull request by LinkIsGrim**
**When merged this pull request will:**
- Title.
- Change initX.sqf files to initX.inc.sqf for ASC exclusion.
- Add `POS_EDGE_SQF` macro, modify `POS_EDGE` macro to work within `QUOTE(...)`.
- Quote a lot of stuff.

### CODE REVIEW

**Comment by mharis001** [comment]
Code looks good. Will verify build when I get a chance.

File: .hemtt/project.toml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+[asc]
+enabled = true
+exclude = [
+    ".inc.sqf",
+    "/dev/"
+]
+
+key_name = "{{prefix}}_{{version}}"
+authority = "{{prefix}}_{{version}}-{{git \"id 8\"}}"
```
Comment (mharis001) [inline .hemtt/project.toml:25]: Does HEMTT still use these?

Comment (LinkIsGrim) [reply .hemtt/project.toml:25]: @BrettMayson

Comment (jonpas) [reply .hemtt/project.toml:25]: No, neither does it use `[header_exts]` section or `releasebuild` field.

Comment (LinkIsGrim) [reply .hemtt/project.toml:25]: Should it be replaced by anything or just dropped?

Comment (jonpas) [reply .hemtt/project.toml:25]: Dropped. HEMTT v1 enforces a standard.

**Comment by mharis001** [comment]
I think this is good. Just want to go through the changes line by line (when I find the time) before we switch to using HEMTT for release builds.

**Review by mharis001 — APPROVED** [review]

**Pull request merged**